Purchases/Logging/RCLoggerBridge.m

```diff
@@ -0,0 +1,51 @@
+//
+//  RCLogger.m
+//  Purchases
+//
+//  Created by RevenueCat.
+//  Copyright © 2021 RevenueCat. All rights reserved.
+//
+
+#import "RCLoggerBridge.h"
+#import "RCLogUtils.h"
+
+@import PurchasesCoreSwift;
+
+@implementation RCLoggerBridge
+
++ (void)setLogLevel:(RCLogLevel)logLevel {
+    switch (logLevel) {
+        case RCLogLevelDebug:
+            RCLog.logLevel = RCInternalLogLevelDebug;
```

Comment:
curious about the need for a separate internal log level?

Reply:
the problem to solve here is that logLevel is defined in the swift module, but used and exposed publicly by the objc module. 

So the idea here is to have an internal one used for swift, and replicate its structure for the public one in objc. 

An alternative approach is to define the exact same interface in a header file in objective c as what we have in swift. This would allow us to have a single enum. 

This is what we do for Transaction.swift ([objc here](https://github.com/RevenueCat/purchases-ios/blob/develop/Purchases/Public/RCTransaction.h), https://github.com/RevenueCat/purchases-ios/blob/develop/PurchasesCoreSwift/Transaction.swift), but I think there were some issues doing this for enums, where you'd get errors when archiving because they'd be identified as duplicate symbols. 

I'm going to take another look to see if matching the exact declarations would work, but going with the duplicate `internal` enum is safer in that we won't face the duplicate symbols issue when archiving

Reply:
ahh ok i think i get it. one thought i had would be setting the private enum value as a val on the public enum, then at least you can use property access to avoid the switches?

Reply:
could you elaborate? do you mean just setting them to the same Int value?